doc/dev_guide/internals/replication/orphan.rst

```diff
@@ -53,16 +50,13 @@ The following situations are possible.
 Here ``box.cfg{}`` is being called for the first time.
 A replica is joining but no replica set exists yet.
 
-    1.  Set status to 'orphan'.
-    2.  Try to connect to all nodes from ``box.cfg.replication``,
-        or to the number of nodes required by
-        :ref:`replication_connect_quorum <cfg_replication-replication_connect_quorum>`.
+    1.  Set the status to 'orphan'.
+    2.  Try to connect to all nodes from ``box.cfg.replication``.
         Retrying up to 3 times in 30 seconds is possible because this is bootstrap,
         :ref:`replication_connect_timeout <cfg_replication-replication_connect_timeout>`
```

Comment:
These two lines seem outdated. I don't even know what "Retrying up to 3 times in 30 seconds" means.
The replica tries to connect for `replication_connect_timeout` seconds and retries each `replication_timeout` seconds if needed.

I'll probably open another ticket for this, because the whole page feels somewhat outdated:
https://github.com/tarantool/doc/issues/3438

Reply:
thanks, fixed